dashboard/app/views/home/_gallery.html.haml

```diff
@@ -1,9 +1,30 @@
+-show_heading = local_assigns[:show_heading]
+
 .course_progress
-  .row#lesson
-    .span12#heading
-      %h3.whitetext{style: 'margin: 0'}
-        = t("home.your_gallery")
+  -if show_heading
+    .row#lesson
+      .span12#heading
+        %h3.whitetext{style: 'margin: 0'}
+          = t("home.your_gallery")
   .row#stats
     .span12{style: 'margin-left: 10px;'}
       %div.gallery#gallery
         = render partial: 'home/gallery_content'
+
+:javascript
+  $(document).ready(function() {
```

Comment:
I think that the right way of doing this according to how Paul set everything up would be to have an entry point for home/index.haml, which would include the inlined JS in that file, as well as the inlined JS here.

Given that this is just moving existing inlined JS, I'm not sure I care quite as much as if it were new JS. It also seems like there's too much indenting on this JS.